docs/confproject.rst

```diff
@@ -90,3 +90,61 @@ On Nginx, the following directive in your site configuration will allow "pretty"
    location /{
          rewrite ^/(.*)$ /index.php?c=$1 last;
    }
+
+Laravel Valet Driver
+^^^^^
+
```

Comment:
For those like me who don't know `Laravel valet`,
you could add in what context it is interesting to use it.

Otherwise, the user may feel that he needs to go through this step.